pkgs/development/compilers/llvm/5/clang/default.nix

```diff
@@ -4,7 +4,6 @@
 }:
 
 let
-  gcc = if stdenv.cc.isGNU then stdenv.cc.cc else stdenv.cc.cc.gcc;
```

Comment:
This stuff is needed at least for for native gcc, because we wish to use `libgcc` rather than `compiler-rt` across the board. (Also there is a were `crt0.o`-like odds and ends that `compiler-rt` never bothered to provide!)

Reply:
Do you think we can use libgcc instead of gcc? Part of my concern is gcc is huge to build.

Reply:
Part of the issue is things like WebAssembly don't support libgcc. wasm-cross doesn't seem to need it when you set rtlib=compiler-rt.

Reply:
I think it depends per target. For WASM one probably doesn't need any GCC at all. For Linux, it probably needs some of them for compat with GCC-built binaries.

Reply:
I believe compiler-rt is designed to replace what Clang uses libgcc for.

Reply:
Yes @dtzWill beat me to linking https://reviews.llvm.org/D28791